src/libratbag-data.c

```diff
@@ -269,6 +283,98 @@ init_data_steelseries(struct ratbag *ratbag,
 	}
 }
 
+static void
+init_data_asus(struct ratbag *ratbag,
+		GKeyFile *keyfile,
+		struct ratbag_device_data *data)
+{
+	const char *group = "Driver/asus";
+	int num, i;
+	GError *error = NULL;
+	_cleanup_(freep) char *str = NULL;
+	_cleanup_(g_strfreevp) char **strv = NULL;
```

Comment:
Use a separate variable for each `g_key_file_get_string_list` call or there will be a memory leak.